pkg/init/Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM alpine:3.5
-
-RUN \
-  apk --no-cache update && \
-  apk --no-cache upgrade -a && \
-  apk --no-cache add \
-  && rm -rf /var/cache/apk/*
-
+# Use sha256 here to get a fixed version
+FROM alpine:edge@sha256:99588bc8883c955c157d18fc3eaa4a3c1400c223e6c7cabca5f600a3e9f8d5cd
```

Comment:
hmm, not ideal as this will be out of date (new `edge` images are not pushed very often, linuxkit/alpine does an upgrade). On the other hand, will remove `alpine` from here soon so lets leave this for now.